frontend/src/app/components/filters/query-filter/query-filter.component.html

```diff
@@ -68,7 +75,7 @@
                            [shouldFocus]="shouldFocus"
                            [filter]="filter"></filter-string-value>
 
-      <filter-boolean-value *ngSwitchCase="'[1]Boolean'"
+      <filter-boolean-value *ngSwitchCase="'[1]Boolean'"  hidden
```

Comment:
Why did you add a `hidden` here instead of removing this value alltogether?

Reply:
Hi @oliverguenther , thanks for your feedback, i agree with you to create a separate component to be used for the whole project. we can reduce the size of it to be more user-friendly.

When i removed filter-boolean-value component, i have an error in  filter-toggled-multiselect-value.component.ts file where there is function    

 private fetchAllowedValues() {
    if ((this.filter.currentSchema!.values!.allowedValues as CollectionResource)['$load']) {
      this.loadAllowedValues();
    } else {
      this.availableOptions = (this.filter.currentSchema!.values!.allowedValues as HalResource[]);
    }
  }
,   (this.filter.currentSchema!.values!.allowedValues as CollectionResource) is null at the time of selecting a boolean filter. 

filter-toggled-multiselect-value component is located after filter-boolean-value component in query-filter.component.html .

I just set hidden to it to ignore this error at the moment.